assets/training/model_management/src/azureml/model/mgmt/utils/exceptions.py

```diff
@@ -27,6 +27,7 @@ class ModelImportErrorStrings:
     VM_NOT_SUFFICIENT_FOR_OPERATION = "VM not sufficient for {operation} operation. Details: [{details}]"
     CMD_EXECUTION_ERROR = "Error in executing command. Error: [{error}]"
     MODEL_ALREADY_EXISTS = "Model with name {model_id} already exists in registry {registry} at {url}"
+    UNSUPPORTED_TASK_TYPE = ("Unsupported task type {task_type} provided. Supported task types are {supported_tasks}.")
```

Comment:
(nit) Are the opening closing '(' ')' necessary?

Reply:
nope, will remove